.travis.yml

```diff
@@ -16,6 +18,15 @@ python:
 - '3.6'
 matrix:
   include:
+  - os: osx
+    language: generic
+    env: NMODE=nikola
+    python: '3.6'
+    before_install:
+    - brew upgrade pyenv
```

Comment:
Why not `brew install python3; brew link python3`? I’m not a fan of pyenv myself, and the Homebrew python is pretty good. Plus, it doesn’t need to be updated to track Python patch versions (and minor versions, too)